examples/client.rs

```diff
@@ -134,6 +133,13 @@ fn main() {
     // Create a QUIC connection and initiate handshake.
     let mut conn = quiche::connect(url.domain(), &scid, &mut config).unwrap();
 
+    info!(
+        "connecting to {:} from {:} with scid {}",
```

Comment:
how about sticking in the QUIC version and ALPN in here - just so we can quickly spot obvious mistmatches?

Reply:
This is pre-handshake, so ALPN is always static and version is whatever the user sets with `--wire-version` (or the default value to trigger negotiation), so not sure this is particularly useful. Though it's easy enough to add.

Reply:
yeah. I guess that information might have been useful if the client examples would have needed to be more widely used but now we have curl + browser(s).